stencil-library/src/components/card-form/card-form.stories.ts

```diff
@@ -95,6 +95,25 @@ Basic.args = {
   'single-line': false,
 };
 
+export const SingleLine = Template.bind({});
+SingleLine.args = {
+  'single-line': true,
+}
+
+export const Embedded = Template.bind({});
+Embedded.decorators = [
```

Comment:
I don't think we want the "Embedded" stories, but instead to show the components on their own

Reply:
Oh OK I did not add the `Embedded` stories - I think that Github shows that I added it because I cleared up a merge conflict around these lines. 

The `Embedded` stories defined on line 103 already exists in the main branch. I only added lines 98-101 and had to fix a merge conflict here.